Corona-Warn-App/src/main/java/de/rki/coronawarnapp/bugreporting/censors/dcc/DccQrCodeCensor.kt

```diff
@@ -77,7 +79,7 @@ class DccQrCodeCensor @Inject constructor() : BugCensor {
         dates += validUntilFormatted
 
         validUntil?.let {
-            dates += it.toShortDayFormat()
+            dates += it.format(DateTimeFormatter.ofLocalizedDate(FormatStyle.SHORT))
```

Comment:
Why not add this to JavaTimeAndDateExtenstions?

Reply:
Well, I prefer to use the standard way directly , I will had to look into the implementation of these functions to see what format is uses.